spec.html

```diff
@@ -38407,7 +38523,7 @@ <h1>JSON.parse ( _text_ [ , _reviver_ ] )</h1>
         1. Let _jsonString_ be ? ToString(_text_).
         1. [id="step-json-parse-validate"] Parse ! UTF16DecodeString(_jsonString_) as a JSON text as specified in ECMA-404. Throw a *SyntaxError* exception if it is not a valid JSON text as defined in that specification.
         1. Let _scriptString_ be the string-concatenation of *"("*, _jsonString_, and *");"*.
-        1. [id="step-json-parse-parse"] Let _completion_ be the result of parsing and evaluating ! UTF16DecodeString(_scriptString_) as if it was the source text of an ECMAScript |Script|. The extended PropertyDefinitionEvaluation semantics defined in <emu-xref href="#sec-__proto__-property-names-in-object-initializers"></emu-xref> must not be used during the evaluation.
+        1. [id="step-json-parse-parse"] Let _completion_ be the result of parsing and evaluating ! UTF16DecodeString(_scriptString_) as if it was the source text of an ECMAScript |Script|.
```

Comment:
I think it's worth keeping a note here that the this evaluation differs from semantics used outside of JSON.parse.

Reply:
Would [``The extended <emu-xref href="#sec-object-initializer-runtime-semantics-propertydefinitionevaluation">PropertyDefinitionEvaluation</emu-xref> semantics defined in step <emu-xref href="#step-object-initializer-runtime-semantics-propertydefinitionevaluation-__proto__-property-name"></emu-xref> of <emu-grammar>PropertyDefinition : PropertyName `:` AssignmentExpression</emu-grammar> must not be used during the evaluation.``](https://ci.tc39.es/preview/tc39/ecma262/pull/2175#step-json-parse-parse) work?